.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release Charts
```

Comment:
Can you please rename this file to `helm-chart-release.yml` please?

Reply:
@diemol done